src/amqp_selective_consumer.erl

```diff
@@ -1,4 +1,4 @@
-%% The contents of this file are subject to the Mozilla Public Licensbe
+%% The contents of this file are subject to the Mozilla Public License
```

Comment:
This is a change to a license blob, but in principle, it's correct.

Reply:
Yes, that's a good catch and we can accept it 👍